.circleci/local_publish_helpers.sh

```diff
@@ -279,3 +279,19 @@ function runE2eTest {
         yarn run e2e --force-exit --detectOpenHandles --maxWorkers=3 $TEST_SUITE
     fi
 }
+
+function checkPackageVersionsInLocalNpmRegistry {
+    cli_internal_version=$(npm view @aws-amplify/cli-internal version)
+    cli_version=$(npm view @aws-amplify/cli version)
+
+    echo "@aws-amplify/cli-internal version: $cli_internal_version"
+    echo "@aws-amplify/cli version: $cli_version"
+
+    if [[ $cli_internal_version != $cli_version ]]; then
+        echo "Versions did not match."
+        echo "Manual fix: add a commit with a 'feat' prefix that touches the cli-npm package to bump its version."
```

Comment:
There are edge cases there where somebody could have used `feat!` in cli-internal. 
```suggestion
        echo "Manual fix: add a proper conventional commit that touches the amplify-cli-npm package to correct its version. For example https://github.com/aws-amplify/amplify-cli/commit/6f14792d1db424aa428ec4836fed7d6dd5cccfd0"
```